ccd-definition/ComplexTypes/BusinessProcess.json

```diff
@@ -0,0 +1,24 @@
+[
+  {
+    "ID": "BusinessProcess",
+    "ListElementCode": "processInstanceId",
+    "FieldType": "Text",
+    "ElementLabel": "Camunda process instance Id",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "BusinessProcess",
+    "ListElementCode": "status",
+    "FieldType": "FixedList",
+    "FieldTypeParameter": "BusinessProcessStatus",
+    "ElementLabel": "Camunda business process status",
+    "SecurityClassification": "Public"
+  },
+  {
+    "ID": "BusinessProcess",
+    "ListElementCode": "taskId",
+    "FieldType": "Text",
+    "ElementLabel": "Last successfully processed task activity id",
+    "SecurityClassification": "Public"
+  }
```

Comment:
why taskid? this will be a different id every time the external task is triggered

Reply:
it's activity Id, named as Task Id